### PR TITLE
try combining coverage with coveralls

### PR DESCRIPTION
**Pull request by ChristopherChudzicki**
Trying: https://docs.coveralls.io/parallel-build-webhook not from fork

### CODE REVIEW

**Comment by jolyonb** [comment]
Lol, you killed coveralls...

File: .travis.yml [hunk 1]
```diff
@@ -7,8 +7,11 @@ matrix:
     install:
       - pip install -r requirements.txt
       - pip install python-coveralls
+    env: COVERALLS_PARALLEL=true
     script:
       - pytest --cov=mitxgraders --cov-report=term-missing
+    notifications:
```
Comment (jolyonb) [inline .travis.yml]: Try removing the notification and webhook lines. We're using https://github.com/z4r/python-coveralls/tree/master to run coveralls, and it has parallel built in (see readme file at that repo).

**Comment by jolyonb** [comment]
Ok, that didn't work. I need to head out, but perhaps this will be helpful? https://github.com/lemurheavy/coveralls-public/issues/1224

File: .travis.yml (hunk 1 above)
Comment (ChristopherChudzicki) [reply .travis.yml]: I've been looking at that and related threads for a while 🙍‍♂ 

This seems promising but more complicated: https://github.com/travis-ci/beta-features/issues/42

Trouble with just using the `combine` command from `coverage` is that it needs files written by `pytest`, which aren't shared between Travis stages, so they need to be uploaded somewhere (AWS S3 in the linked issue.)

**Comment by ChristopherChudzicki** [comment]
I've been looking at that and related threads for a while 🙍‍♂

This seems promising but more complicated: travis-ci/beta-features#42

Trouble with just using the combine command from coverage is that it needs files written by pytest, which aren't shared between Travis stages, so they need to be uploaded somewhere (AWS S3 in the linked issue.)

**Pull request closed without merging**